dev/com.ibm.ws.crypto.passwordutil/test/com/ibm/websphere/crypto/PasswordUtilTest.java

```diff
@@ -133,7 +130,9 @@ public void testUtil() {
 
     @Test
     public void testAESEncoding() throws Exception {
-        assertEquals("The password was not decoded correctly", "alternatepwd", PasswordUtil.decode("{aes}AEmVKa+jOeA7pos+sSfpHNmH1MVfwg8ZoV29iDi6I0ZGcov6hSZsAxMhFr91jTSBYQ=="));
+        //TODO remove beta setting once AES-256 is GA
+        System.setProperty("com.ibm.ws.beta.edition", "true");
```

Comment:
Should a try/finally unset the beta system property so it isn't set for future tests?

Reply:
Or better yet a test to show it failing if beta is not set so that your end up testing the exception path as well.  When you remove the beta guard it will make the unit test fail showing tests that need to be updated.